package.json

```diff
@@ -10,17 +10,17 @@
     "registry": "https://npm.pkg.github.com/sohohouse"
   },
   "dependencies": {
-    "babel-eslint": "^7.2.3",
```

Comment:
babel-eslint is no longer supported, it is recommended to use "@babel/eslint-parser"